src/Eclo.DataAccess/Repositories/Products/ProductDetailSizeRepository.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Dapper;
+using Eclo.DataAccess.Interfaces.Products;
+using Eclo.Domain.Entities.Discounts;
+using Eclo.Domain.Entities.Products;
+
+namespace Eclo.DataAccess.Repositories.Products;
+
+public class ProductDetailSizeRepository : BaseRepository, IProductDetailSizeRepository
+{
+    public async Task<long> CountAsync()
+    {
+        try
+        {
+            await _connection.OpenAsync();
+            string query = "SELECT COUNT(*) FROM product_detail_sizes";
+            var result = await _connection.QuerySingleAsync<long>(query);
+            return result;
+        }
+        catch
+        {
+            return 0;
+        }
+        finally
+        {
+            await _connection.CloseAsync();
+        }
+    }
+
+    public async Task<int> CreateAsync(ProductDetailSize entity)
+    {
+        try
+        {
+            await _connection.OpenAsync();
+            string query = "INSERT INTO public.product_detail_sizes (product_detail_id, size, quantity, created_at, updated_at) " +
+                "VALUES (@ProductDetailId, @Size, @Quantity, @CreatedAt, @UpdatedAt);";
+            var result = await _connection.ExecuteAsync(query, entity);
```

Comment:
var result variable tepa va pastki kodlardan ajralib tursin.

Reply:
All lines like this have been changed or rewritten